egs/wsj/s5/steps/nnet3/chain/get_egs.sh

```diff
@@ -488,7 +488,7 @@ if [ $stage -le 6 ]; then
     # 'storage' directory.
     rm cegs_orig.*.ark 2>/dev/null
   )
-  if [ $archives_multiple -gt 1 ]; then
+  if ! $generate_egs_scp && [ $archives_multiple -gt 1 ]; then
```

Comment:
Vimal, can you explain why this is necessary?  I'm having trouble understanding the script now.

Reply:
Forget it, I figured it out.